ci-operator/step-registry/ibmcloud/deprovision/kms-key/ibmcloud-deprovision-kms-key-commands.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output
+    region="${LEASED_RESOURCE}"
+    rg=$1
+    export region
+    "${IBMCLOUD_CLI}" config --check-version=false
+    echo "Try to login to ${rg}..."
+    "${IBMCLOUD_CLI}" login -r ${region} -g ${rg} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+function run_command_with_retries() {
+  cmd="$1"
+  retries="$2"
+  interval="$3"
+
+  if [ X"$retries" == X"" ]; then
+      retries=20
+  fi
+
+  if [ X"$interval" == X"" ]; then
+      interval=30
+  fi
+
+  set +o errexit
+  output=$(eval "$cmd"); ret=$?
+  try=1
+
+  # avoid exit with "del Resource groups with active or pending reclamation instances can't be deleted"
+  while [ X"$ret" != X"0" ] && [ $try -lt $retries ]; do
+      sleep $interval
+      output=$(eval "$cmd"); ret=$?
+      try=$(expr $try + 1)
+  done
+  set -o errexit
+
+  if [ X"$try" == X"$retries" ]; then
+      return 2
+  fi
+  echo "$output"
+  return 0
+}
+
+resource_group_file=${SHARED_DIR}/ibmcloud_resource_group
+
+RESOURCE_GROUP=$(cat ${resource_group_file})
+echo "ResourceGroup: ${RESOURCE_GROUP}"
+ibmcloud_login ${RESOURCE_GROUP}
+
+
+kp_file=${SHARED_DIR}/ibm_kp
+key_file=${SHARED_DIR}/ibm_kpKey.yaml
+
+if [ -f ${kp_file} ]; then
+    id=$(cat ${kp_file})
+    crn=$(yq-go r ${key_file} 'platform.ibmcloud.defaultMachinePlatform.bootVolume.encryptionKey')
```

Comment:
Once resolved https://github.com/openshift/release/pull/46712/files#r1432520210, you can get the key id easily, not have to rely on `key_file`